services/proxy/pkg/config/config.go

```diff
@@ -24,28 +24,29 @@ type Config struct {
-	InsecureBackends      bool                 `yaml:"insecure_backends" env:"PROXY_INSECURE_BACKENDS" desc:"Disable TLS certificate validation for all HTTP backend connections." introductionVersion:"pre5.0"`
-	BackendHTTPSCACert    string               `yaml:"backend_https_cacert" env:"PROXY_HTTPS_CACERT" desc:"Path/File for the root CA certificate used to validate the server’s TLS certificate for https enabled backend services." introductionVersion:"pre5.0"`
-	AuthMiddleware        AuthMiddleware       `yaml:"auth_middleware"`
-	PoliciesMiddleware    PoliciesMiddleware   `yaml:"policies_middleware"`
-	CSPConfigFileLocation string               `yaml:"csp_config_file_location" env:"PROXY_CSP_CONFIG_FILE_LOCATION" desc:"The location of the CSP configuration file." introductionVersion:"6.0.0"`
-	Events                Events               `yaml:"events"`
-	ClaimSpaceManagement  ClaimSpaceManagement `yaml:"claim_space_management"`
+	RoleQuotas                map[string]uint64    `yaml:"role_quotas"`
+	Policies                  []Policy             `yaml:"policies"`
+	AdditionalPolicies        []Policy             `yaml:"additional_policies"`
+	OIDC                      OIDC                 `yaml:"oidc"`
+	ServiceAccount            ServiceAccount       `yaml:"service_account"`
+	RoleAssignment            RoleAssignment       `yaml:"role_assignment"`
+	PolicySelector            *PolicySelector      `yaml:"policy_selector"`
+	PreSignedURL              PreSignedURL         `yaml:"pre_signed_url"`
+	AccountBackend            string               `yaml:"account_backend" env:"PROXY_ACCOUNT_BACKEND_TYPE" desc:"Account backend the PROXY service should use. Currently only 'cs3' is possible here." introductionVersion:"pre5.0"`
+	UserOIDCClaim             string               `yaml:"user_oidc_claim" env:"PROXY_USER_OIDC_CLAIM" desc:"The name of an OpenID Connect claim that is used for resolving users with the account backend. The value of the claim must hold a per user unique, stable and non re-assignable identifier. The availability of claims depends on your Identity Provider. There are common claims available for most Identity providers like 'email' or 'preferred_username' but you can also add your own claim." introductionVersion:"pre5.0"`
+	UserCS3Claim              string               `yaml:"user_cs3_claim" env:"PROXY_USER_CS3_CLAIM" desc:"The name of a CS3 user attribute (claim) that should be mapped to the 'user_oidc_claim'. Supported values are 'username', 'mail' and 'userid'." introductionVersion:"pre5.0"`
+	MachineAuthAPIKey         string               `yaml:"machine_auth_api_key" env:"OCIS_MACHINE_AUTH_API_KEY;PROXY_MACHINE_AUTH_API_KEY" desc:"Machine auth API key used to validate internal requests necessary to access resources from other services." introductionVersion:"pre5.0" mask:"password"`
```

Comment:
I have never seen before a `mask` key such as ` mask:"password"` with envvars. It maybe here for longer...
We also did not document it in the dev docs. What does this key do?

Reply:
Clarified with @kobergj , no issue :sweat_smile: